askbot/media/js/post.js

```diff
@@ -2479,13 +2488,15 @@ Comment.prototype.setContent = function (data) {
     }
 
     if (this._is_convertible) {
-        if (this._convert_link !== undefined) {
-            this._convert_link.dispose();
```

Comment:
so here dispose basically was removing the form from DOM, and on line 2493 we were trying to replace the contents of that form with correct data which wasn't very helpful